types/emscripten/index.d.ts

```diff
@@ -114,9 +117,51 @@ declare namespace FS {
         node: FSNode;
     }
 
-    interface FSStream {}
-    interface FSNode {}
-    interface ErrnoError {}
+    interface Mount {
+        type: Emscripten.FileSystemType;
+        opts: object;
+        mountpoint: string;
+        mounts: Mount[];
+        root: FSNode;
+    }
+
+    class FSStream {
+        shared?: {
```

Comment:
It kind of looks like this is not meant to be public API, since it’s just backing data for the other getters/setters. At least looks like it shouldn’t be optional.

Reply:
Do you think it would be appropriate to remove it or to mark it as protected?

Reply:
Remove

Reply:
Removed the field in a2b5ba1